kernel-devel.sh

```diff
@@ -0,0 +1,8 @@
+package: kernel-devel
+version: "1.0"
+
+system_requirement_missing: "Please install kernel-devel pciutils-devel kmod-devel"
+system_requirement: ".*"
+system_requirement_check: "ls /usr/src/kernels/*/include/linux/kernel.h > /dev/null 2>&1 && ls /usr/include/libkmod.h  > /dev/null 2>&1 && ls /usr/include/pci/pci.h > /dev/null 2>&1"
```

Comment:
I agree with @ktf, but we can keep it like this for now given the scope. I assume the paths are correct on CC7, right?